app/controllers/dish_ingredients_controller.rb

```diff
@@ -0,0 +1,3 @@
+class DishIngredientsController < ApplicationController
```

Comment:
This would be a great place to handle the functionality for adding an ingredient to a dish! This is where we could add a `create` action and in that create action, add everything that you currently have in the update action in the dishes_controller